### DISCUSSION TITLE
Mac系统下Alt+x 的映射是某类字符,而不是<A+x>或者<M+x>

### DISCUSSION THREAD

**Discussion by est7** [nshen/learn-neovim-lua#162] [Q&A]
按下Alt+x得到是:≈,按下Alt+j得到的是∆,这样nvim里的alt+的映射全都不管用了...请问大家有什么办法屏蔽掉alt+字符的转化

**Comment by nshen** [comment]
<img width="741" alt="image" src="https://user-images.githubusercontent.com/181506/224606813-ebb3cf9a-bc8e-4df5-9e1f-f0ac122cb9c1.png">

**Comment by est7** [comment]
感谢帮助,我在提问题尝试了这个办法,还是不行,不知道还有可能是哪里的问题?

**Reply by nshen** [reply]
你是怎么映射的，直接映射符号？
